docs/UserGuide.md

```diff
@@ -71,9 +71,16 @@ Examples:
 
 Shows a list of all items in the inventory.
 
-Format: `list [l/LOCATION]`
+Format: `list [LOCATION]`
 
 * You can view all items in the inventory by typing 'list' without specifying location.
+* The search is case-insensitive. e.g 'room' will match 'Room'.
+* The order of keywords does not matter. e.g. 'Room Living' will match 'Living Room'.
+* Only location will be searched.
+* Only full words will be matched e.g. 'Room' will not match 'Bedroom'.
+* Location matching uses each word in the String to do 'AND' search e.g. 'Room' will match 'Living room' but 'Living room 1' will not match 'Living room'
```

Comment:
It May be too technical for a user guide

Reply:
I think it is fine. I follow the AB3 format, but we can always revisit the User guide and developer guide along the way. :D